client/data/blogging-prompt/use-blogging-prompts.ts

```diff
@@ -27,12 +28,12 @@ export const useBloggingPrompts = (
 	page: number
 ): UseQueryResult< BloggingPrompt[] | null > => {
 	const today = moment().format( 'YYYY-MM-DD' );
-
+	const isBloganuary = isEnabled( 'bloganuary' );
 	return useQuery( {
-		queryKey: [ 'blogging-prompts', today + '-' + page ],
+		queryKey: [ 'blogging-prompts', today + '-' + page, isBloganuary ],
 		queryFn: () =>
 			wp.req.get( {
-				path: `/sites/${ siteId }/blogging-prompts?number=${ page }&from=${ today }`,
+				path: `/sites/${ siteId }/blogging-prompts?number=${ page }&from=${ today }&is_bloganuary=${ isBloganuary }`,
```

Comment:
may be easier/cleaner to use addQueryArgs for these, and that could keep the is_bloganuary=false from appearing the rest of the year.